external/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI += "file://0001-eglfs-add-a-default-framebuffer-to-NVIDIA-eglstreams.patch \
+SRC_URI:tegra += "file://0001-eglfs-add-a-default-framebuffer-to-NVIDIA-eglstreams.patch \
```

Comment:
This should use `:append:tegra =` as well

Reply:
right, I will change it to `':append:tegra = " '`